MediaGalleryUi/view/adminhtml/web/template/grid/masonry.html

```diff
@@ -0,0 +1,17 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<div data-role="grid-wrapper" class="masonry-image-grid" attr="'data-id': containerId">
+    <div class="masonry-image-column media-gallery-image-column" repeat="foreach: rows, item: '$row'">
```

Comment:
I don't think it's necessary to override the masonry component and template only to add the class. I'd propose to use '.media_gallery-index-index' parent selector instead for now, and later (when media gallery will be moved to slide panel) adjust it to slide panel selector

Reply:
ok